pyproject.toml

```
@@ -75,12 +77,13 @@ dev-mypy = [
   "frequenz-client-dispatch[dev-mkdocs,dev-noxfile,dev-pytest]",
   "grpc-stubs == 1.53.0.5",
   "types-protobuf == 5.26.0.20240422",
+  "frequenz-client-dispatch[cli]",
 ]
 dev-noxfile = ["nox == 2024.4.15", "frequenz-repo-config[lib] == 0.9.2"]
 dev-pylint = [
   "pylint == 3.1.0",
   # For checking the noxfile, docs/ script, and tests
-  "frequenz-client-dispatch[dev-mkdocs,dev-noxfile,dev-pytest]",
+  "frequenz-client-dispatch[dev-mkdocs,dev-noxfile,dev-pytest,cli]",
```

Nitpick, keep alphabetical order :grimacing: 
```suggestion
  "frequenz-client-dispatch[cli,dev-mkdocs,dev-noxfile,dev-pytest]",
```